Description: Strip the trailing dot when making a reverse on IPv6 addresses.
Author: Laurent Bigonville <bigon@debian.org>
Bug-Debian: http://bugs.debian.org/397544

---
 wwwroot/cgi-bin/plugins/ipv6.pm |    1 +
 1 file changed, 1 insertion(+)

--- a/wwwroot/cgi-bin/plugins/ipv6.pm
+++ b/wwwroot/cgi-bin/plugins/ipv6.pm
@@ -64,6 +64,7 @@
 	my $query = $resolver->query($reverseip, "PTR");
 	if (! defined($query)) { return; }
 	my @result=split(/\s/, ($query->answer)[0]->string);
+	chop($result[4]); # Remove the trailing dot of the answer.
 	return $result[4];
 	# ----->
 }
